index.html

```diff
@@ -47,7 +47,7 @@
   </p>
 </section>
 <section id="sotd">
-  <p>This document is an early draft and does not reflect the consensus of the TAG.</p>
+  <p>This document has been produced by the W3C Technical Architecture Group (TAG). The TAG approved this finding at its [December 2024 F2F](https://github.com/w3ctag/meetings/tree/gh-pages/2024/12-edinburgh).</p>
```

Comment:
I also see `specStatus: "draft-finding",` above, which should probably change to `"finding"`. With that change made, if you remove this section entirely, do you get good output?